### ISSUE TITLE
README references a top-level libpldm/ directory

### ISSUE THREAD

**Issue by amboar** [openbmc/libpldm#1]
This no-longer exists, and so we should clean it up to reflect the new structure, along with any other affected documentation.

At least the following should be fixed:

* https://github.com/openbmc/libpldm/blame/3896ecc787743360b5b1c03e79aa8a2d0fd1c969/README.md#L41
* https://github.com/openbmc/libpldm/blame/3896ecc787743360b5b1c03e79aa8a2d0fd1c969/README.md#L61
* https://github.com/openbmc/libpldm/blame/3896ecc787743360b5b1c03e79aa8a2d0fd1c969/README.md#L66

**Comment by manojkiraneda** [comment]
hey @amboar , the lines that you mentioned point to openbmc/libpldm 

for example [this](https://github.com/openbmc/libpldm/blame/3896ecc787743360b5b1c03e79aa8a2d0fd1c969/README.md#L66) points to [this](https://github.com/openbmc/libpldm/blob/main/meson.build)

was that not clear in the documentation ?

**Comment by amboar** [comment]
> hey @amboar , the lines that you mentioned point to openbmc/libpldm
> 
> for example [this](https://github.com/openbmc/libpldm/blame/3896ecc787743360b5b1c03e79aa8a2d0fd1c969/README.md#L66) points to [this](https://github.com/openbmc/libpldm/blob/main/meson.build)
> 
> was that not clear in the documentation ?

Right, but the libpldm/ top-level directory isn't something that exists any more. It did when the code was part of openbmc/pldm.git, but we chopped it out. It's now not self-consistent, so we should fix it. I wasn't sure whether there were other issues along these lines, so opened an issue to try to figure that out. If there's not I can send a patch to fix up the bits I linked to above.
